xds/internal/server/conn_wrapper.go

```diff
@@ -58,14 +62,21 @@ type connWrapper struct {
 	deadlineMu sync.Mutex
 	deadline   time.Time
 
+	mu       sync.Mutex
+	st       transport.ServerTransport
+	draining bool
+
 	// The virtual hosts with matchable routes and instantiated HTTP Filters per
-	// route.
-	virtualHosts []xdsresource.VirtualHostWithInterceptors
+	// route, or an error.
+	rc *unsafe.Pointer // *RoutingConfiguration
 }
 
-// VirtualHosts returns the virtual hosts to be used for server side routing.
-func (c *connWrapper) VirtualHosts() []xdsresource.VirtualHostWithInterceptors {
-	return c.virtualHosts
+// RoutingConfiguration returns the RoutingConfiguration to be used for server
+// side routing. If RoutingConfiguration contains error, fail any RPCs on this
+// Conn with status code UNAVAILABLE.
+func (c *connWrapper) RoutingConfiguration() xdsresource.RoutingConfiguration {
```

Comment:
Similar to the comment somewhere else on this PR about this name being too close to other existing names, maybe rename this method appropriately as well. If you end up naming the type `UsableRouteConfig` or `UsableRouteConfiguration` or `MatchableRouteConfig` or whatever, this method name could be the same.

> If RoutingConfiguration contains error, fail any RPCs on this Conn with status code UNAVAILABLE.

Does this comment belong here? This is an internal type with the method being called via an anonymous interface type assertion. Maybe, this needs to be part of the xds server's `Serve` method? Somewhere where this can get to the user.

Reply:
Switched method name to UsableRouteConfiguration. Deleted that part of the docstring comment.